app/archivesspace/archivesspace.controller.js

```diff
@@ -116,8 +123,19 @@ controller('ArchivesSpaceController', ['$scope', '$uibModal', 'Alert', 'Archives
             return node.date_expression;
           },
           note: () => {
-            if (node.notes && node.notes[0]) {
-              return node.notes[0].content;
+            if (node.notes) {
+              let general_notes = node.notes.filter(note => note.type === 'odd');
+              if (general_notes.length > 0) {
+                return general_notes[0].content;
```

Comment:
In the future we might change this, but right now the UI only supports a single note per type.